Qs.Presentation.WebApi/Dockerfile

```diff
@@ -0,0 +1,20 @@
+﻿FROM mcr.microsoft.com/dotnet/aspnet:6.0 AS base
```

Comment:
Don't forget to update the Dockerfile

Reply:
Added docker-compose.yml file. Making it HTTPS compatible depends on I think whether we want to use middleware or not.